ext/js/dictionary/dictionary-importer.js

```diff
@@ -815,8 +815,19 @@ export class DictionaryImporter {
         const results = [];
         for (const file of files) {
             const content = await this._getData(file, new TextWriter());
-            /** @type {unknown} */
-            const entries = parseJson(content);
+            let entries;
+
+            try {
+                /** @type {unknown} */
+                entries = parseJson(content);
+            }
+            catch (error) {
+                if (error instanceof Error) {
+                    let newError = new Error(error.message + `. Dictionary has invalid data in '${file.filename}'`);
+                    console.error(newError);
```

Comment:
In addition, I don't think it's useful logging here and still letting the error propagate. If I decided to log the errors upstream, it would cause the error to be logged twice. You can remove the console.error entirely.

Reply:
Ok. I'll do away with it entirely.